packages/fuels/src/cli/types.ts

```diff
@@ -0,0 +1,118 @@
+import type { DeployContractOptions } from '@fuel-ts/contract';
+
+export enum Commands {
+  build = 'build',
+  deploy = 'deploy',
+  dev = 'dev',
+  init = 'init',
+}
+
+export type CommandEvent =
+  | {
+      type: Commands.build;
+      data: unknown;
+    }
+  | {
+      type: Commands.deploy;
+      data: DeployedContract[];
+    }
+  | {
+      type: Commands.dev;
+      data: unknown;
+    }
+  | {
+      type: Commands.init;
+      data: unknown;
+    };
+
+export type DeployedContract = {
+  name: string;
+  contractId: string;
+};
+
+export type ContractDeployOptions = {
+  contracts: DeployedContract[];
+  contractName: string;
+  contractPath: string;
+};
+
+export type OptionsFunction = (
+  options: ContractDeployOptions
+) => DeployContractOptions | Promise<DeployContractOptions>;
+
+export type UserFuelsConfig = {
+  /** Relative directory path to Forc workspace */
+  workspace?: string;
+  /** Relative directory path to Sway contracts */
```

Comment:
```suggestion
  /** List of relative directory path to Sway contracts */
```
To be consistent with the `predicates` and `scripts` description.